rfc/0054-sdk3-management-apis.md

```diff
@@ -3599,11 +3653,18 @@ interface CollectionSpec {
     String ScopeName();
 
     Duration MaxExpiry();
+    
+    Bool History();
 }
 ```
 
 `MaxExpiry` is the time in seconds for the TTL for new documents in the collection. Omit if not set.
 
+`History` is whether history retention override is enabled on this collection. Omit if not set - not set will default to bucket level setting.
+* On Create/Update this setting should be guarded by the bucket capability of `nonDedupedHistory`
+* If the bucket capability is not supported then a `FeatureNotAvailableException` must be raised.
+  * Suggested supplementary text: "history is unsupported by the server, magma storage engine must be used"
```

Comment:
This message seems a bit ambiguous?  Like is it telling me the problem is the server version, or that I'm not using Magma?

Reply:
I've had 2 stabs at the wording and come up lacking :), any suggestions?